### ISSUE TITLE
Using the fork option makes getting blocks fail

### ISSUE THREAD

**Issue by jrainville** [trufflesuite/ganache-core#472]
## Expected Behavior

Using Ganache-cli using the `--fork` option should work the same as without fork in terms of using Web3.js function like `getBlock` and `getBlockNumber`

## Current Behavior

When using Ganache as follows: 
```
ganache-cli -d --fork https://mainnet.infura.io -p 9545
```
And then connecting an instance of Web3.js to it like so:
```
const web3 = new Web3('http://localhost:9545');
```
Calling functions to get blocks fail.

Eg: Calling 
```
const block = await web3.eth.getBlock(0, true);
// or
const blockNumber = await web3.eth.getBlockNumber();
```
both throw the following error:
```
invalid argument 0: json: cannot unmarshal hex string of odd length into Go value of type common.Hash
```

Doing the same but with 
```
ganache-cli -d -p 9545
```
(no fork) works perfectly

## Steps to Reproduce (for bugs)

1. Start ganache like so: `ganache-cli -d --fork https://mainnet.infura.io -p 9545`
2. In a node REPL or a small node file, do:
```
const Web3 = require('Web3');
const web3 = new Web3('http://localhost:9545');
try {
  const block = await web3.eth.getBlock(0, true);
} catch (e) {
  console.error(e); // this will be thrown
}
```

## Context

Experienced by me and one of our Embark users when trying to test contract using a fork of mainnet in Ganache. 

Right now, we patched Embark to just show the error and lose a tracking functionality because we don't have access to the blocks, but the ideal would be to fix it in Ganache so that block functions are available all the time, even with a fork.

## Your Environment
* Version used: 6.4.3
* NodeJS Version: 10.13.0
* Operating System and version (include distro if Linux): Windows 10 for me and MacOS (I think for the user)

**Comment by seesemichaelj** [comment]
I was able to reproduce the issue in `ganache-cli@6.4.3` and verified this is fixed in the latest version `ganache-cli@6.10.1`. I'm not entirely sure which version fixed this, but if you're running into this issue, please upgrade :)

**Comment by jrainville** [comment]
@seesemichaelj Thanks for the heads up. I'll try this out.
